### DISCUSSION TITLE
Does the wind tab mean Wind Sustained or gusts?

### DISCUSSION THREAD

**Discussion by coolkid0281** [niels747/2D-Weather-Sandbox#75] [Q&A]
I was wondering if The wind tab when you open Soundings means wind sustained or gusts? I'm pretty sure it means sustained tho, but I was wondering if it's that! 
![image](https://user-images.githubusercontent.com/110744575/226192548-85b9aa8a-8ea1-4b70-8f1e-73b6ee063679.png)

**Comment by odvigajlo** [comment]
1. That's not wind tab bruh.
2. It's showing current air velocity converted to miles/hour or meters/hour

**Comment by JustLearninglol** [comment]
It is the sustained wind speed of the simulation, but it can be vertical or horizontal. m/s just means how many meters the wind moves per second. Pressing tab lets you see the direction the wind is going.

**Comment by niels747** [comment]
Yes it is indeed sustained wind speed.
